src/Sylius/Component/Core/spec/Model/ChannelPricingSpec.php

```diff
@@ -49,4 +50,41 @@ function its_price_is_mutable()
         $this->setPrice(1000);
         $this->getPrice()->shouldReturn(1000);
     }
+
+    function it_does_not_have_original_price_by_default()
+    {
+        $this->getOriginalPrice()->shouldReturn(null);
+    }
+
+    function its_original_price_should_be_mutable()
```

Comment:
Avoid word ``should`` in spec names -> ``its_original_price_is_mutable`` would be sufficient